src/senaite/core/content/senaitesetup.py

```diff
@@ -78,9 +78,25 @@ class ISetupSchema(model.Schema):
         required=False,
     )
 
+    immediate_results_entry = schema.Bool(
+        title=_(u"Immediate results entry"),
+        description=_(
+            "Allow the user to directly enter results after sample "
+            "creation, e.g. to enter field results immediately, or lab "
+            "results, when the automatic sample reception is activated."
+        ),
+    )
+
     ###
     # Fieldsets
     ###
+    model.fieldset(
+        "sampling",
```

Comment:
I suggest a "results_entry" fieldset, with label "Results entry". I am sure there are other settings that fit better with this fieldset "schemata" than the one they are currently assigned to.